src/components/molecules/azlist/AZList.scss

```diff
@@ -0,0 +1,31 @@
+.azlist_container .browse {
```

Comment:
regroup/nest azlist_container and contentzone styles

Reply:
so this is a mock component - was just a placeholder